Assignment/Assignment.vcxproj

```diff
@@ -127,11 +127,11 @@
     </Link>
   </ItemDefinitionGroup>

```

Comment:
Tic Tac toe is not a library. It is a standalone program. Header file is not required.